recipes/uvloop/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "uvloop" %}
+{% set version = "0.8.1" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "923b6e4982aea14a23565e6e62335a691b7a939bcc6802fbafa2a146ee3e8ac3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_value }}
+
+build:
+  number: 0
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+  skip: True  # [py2k or win]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+  run:
+    - python
+
+test:
+  imports:
+    - uvloop
+
+about:
+  home: http://github.com/MagicStack/uvloop
+  # Multiple licenses! : Apache 2.0, MIT
+  license: MIT
```

Comment:
Could you add the corresponding `license_file: LICENSE-MIT`

Reply:
Added